tests/test_configure.py

```
@@ -762,36 +765,38 @@ def check_delays(exp: TrackingExporter, min_delay: float, max_delay: float) -> N
 
-def test_configure_service_version(tmp_path: str) -> None:
-    request_mocker = requests_mock.Mocker()
-    request_mocker.get(
-        'https://logfire-api.pydantic.dev/v1/info',
-        json={'project_name': 'myproject', 'project_url': 'fake_project_url'},
-    )
-
+def test_configure_service_version(config_kwargs: dict[str, Any], exporter: TestExporter, tmp_path: str) -> None:
     import subprocess
 
     git_sha = subprocess.check_output(['git', 'rev-parse', 'HEAD']).decode('ascii').strip()
```

Ruff (0.15.17)

[error] Starting a process with a partial executable path

(S607)